recipes-wpe/wpe/wpe_0.1.bb

```diff
@@ -186,4 +186,7 @@ RDEPS_MEDIASOURCE += "${RDEPS_EXTRA}"
 RDEPS_VIDEO += "${RDEPS_EXTRA}"
 RDEPS_WEBAUDIO += "${RDEPS_EXTRA}"
 
-RRECOMMENDS_${PN} += "ca-certificates"
+RRECOMMENDS_${PN} += " \
+    ca-certificates \
+    ttf-bitstream-vera \
```

Comment:
The browser doesn't work without fonts. "ttf-bitstream-vera" is currently pulled in via the packagegroup-ml-wpe package group, so if you use that package group you would have the fonts already. This change just adds some clarity and helps those who don't use the package group.